cyberdrop_dl/crawlers/_fluid_player.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar, NamedTuple
+
+from cyberdrop_dl.crawlers.crawler import Crawler
+from cyberdrop_dl.data_structures.mediaprops import Resolution
+from cyberdrop_dl.utils import css, open_graph
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from bs4 import BeautifulSoup
+
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selector:
+    VIDEO_SRC = ".gif-video, #main_video source"
+    COLLECTION_TITLE = "h2.object-title"
+    SEARCH_VIDEOS = "div.list-videos a.popito"
```

Comment:
This selector works for all sites
```suggestion
    SEARCH_VIDEOS = "div.list-videos .data-video-id > a"
```